tests/phpunit/tests/image/functions.php

```diff
@@ -545,41 +552,49 @@ public function test_crop_setting_for_pdf() {
 
 		$this->assertNotEmpty( $attachment_id );
 
+		$metadata = wp_generate_attachment_metadata( $attachment_id, $test_file );
+
+		$temp_dir = get_temp_dir();
+
 		$expected = array(
-			'sizes' => array(
+			'sizes'    => array(
 				'full'      => array(
 					'file'      => 'wordpress-gsoc-flyer-pdf.jpg',
 					'width'     => 1088,
 					'height'    => 1408,
 					'mime-type' => 'image/jpeg',
+					'filesize'  => wp_filesize( $temp_dir . 'wordpress-gsoc-flyer-pdf.jpg' ),
```

Comment:
One nice thing about using `wp_filesize` we can use the filters in this function to mock the size of the file if we want.